classes/game.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'item'
+require 'date'
+
+class Game < Item
+  attr_reader :multiplayer, :last_played
+
+  def initialize(multiplayer, last_played, publish_date)
+    super(publish_date)
+
+    @multiplayer = multiplayer
+    @last_played = last_played
+    move_to_archive
+  end
```

Comment:
- [ ] I get the following error when adding a new game, kindly fix this issue.
![image](https://user-images.githubusercontent.com/81932478/195997449-0033029a-1d6e-4b06-aea0-771b217f205f.png)

Reply:
Thank you. All issues have fixed